tools/run-bidi-server.mjs

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {spawn, spawnSync} from 'child_process';
+import {mkdirSync, createWriteStream} from 'fs';
+import {join, basename, dirname} from 'path';
+
+import {hideBin} from 'yargs/helpers';
+import yargs from 'yargs';
+
+process.chdir(dirname(dirname(process.argv[1])));
+
+function log(message) {
+  // eslint-disable-next-line no-console
+  console.log(`(${basename(process.argv[1])}) ${message}`);
+}
+
+const argv = yargs(hideBin(process.argv))
+  .usage(
```

Comment:
Can `yargs` generate the `usage` string automatically? It was always a chore to maintain this.

Although, I'm skeptical because these are env vars, not command switches.

Reply:
Yea, because of the env, we can't ;/

Reply:
I would be in favour of converting to proper args in the future (but not in this PR)

Reply:
yargs does generate it automatically